### DISCUSSION TITLE
verbose for debugging

### DISCUSSION THREAD

**Discussion by ji-podhead** [ansible-collections/community.hashi_vault#445] [Q&A] [answered]
Is there a way to debug the lookup of community.hashi_vaul? For instance, can we enable verbose output or similar features to better understand any issues during execution?

**Comment by leogallego** [comment]
Hello @ji-podhead , while we appreciate constructive feedback, let me remind you there is an [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) for those participating in the community and we would also appreciate if you respect it. Please edit your issue to adhere to these policies.

**Comment by briantist** [comment, marked as answer]
@ji-podhead possibilities for debugging and getting output are sometimes limited by what ansible itself can do, since as a plugin we must adhere to the API given. To answer more specifically, I'll need to understand what your issue is: which plugin, your invocation, your error message(s), etc., possibly your ansible and/or Vault configuration as well. Please provide more information about what you're trying to do and where you're running into issues.

**Reply by ji-podhead** [reply]
i just want to get the result of the vault api call

**Reply by briantist** [reply]
It sounds like you want the `raw` return value that's available on some of the plugins and modules: https://docs.ansible.com/ansible/latest/collections/community/hashi_vault/vault_kv2_get_module.html#return-raw

Alternatively you can use the generics (`vault_read`, `vault_write`) which only return the raw API result.

**Reply by ji-podhead** [reply]
ah cool, i guess i should use the right [verbosity level](https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html#cmdoption-ansible-playbook-v) when having trouble with plugins
